src/app/workflow-editor/object-inspector/io-inspector/workflow-io-inspector.component.ts

```diff
@@ -126,39 +201,86 @@ export class WorkflowIOInspectorComponent extends DirectiveBase implements OnIni
 
     ngOnChanges(changes: SimpleChanges): void {
 
-        if (this.form && changes["port"]) {
-            const newIO = changes["port"].currentValue;
-            if (!newIO) { // Happens when you go through wf history to the beginning
-                return;
+        if (this.form) {
+
+            if (changes["port"]) {
+                const newIO = changes["port"].currentValue;
+                if (!newIO) { // Happens when you go through wf history to the beginning
+                    return;
+                }
+
+                this.form.controls["isRequired"].setValue(!newIO.type.isNullable);
+                this.form.controls["id"].setValue(newIO.id);
+                this.form.controls["label"].setValue(newIO.label);
+                this.form.controls["typeForm"].setValue(newIO.type);
+                this.form.controls["symbols"].setValue(newIO.type.symbols || this.initSymbolsList);
+                this.form.controls["description"].setValue(newIO.description);
+                this.form.controls["fileTypes"].setValue(newIO.customProps["sbg:fileTypes"]);
```

Comment:
`WorkflowInputParamterModel` and `WorkflowOutputParameterModel` both have the poperty `fileTypes: string[]`. Use this instead of going into `customProps`.